tgui/packages/tgui/interfaces/Mule.tsx

```diff
@@ -0,0 +1,207 @@
+import { useBackend } from '../backend';
+import {
+  Button,
+  Dropdown,
+  LabeledList,
+  ProgressBar,
+  Section,
+  Stack,
+} from '../components';
+import { BooleanLike } from 'common/react';
+import { Window } from '../layouts';
+import { InterfaceLockNoticeBox } from './common/InterfaceLockNoticeBox';
+
+type Data = {
+  powerStatus: BooleanLike;
+  cellPercent: number | null;
+  load: BooleanLike;
+  locked: BooleanLike;
+  siliconUser: BooleanLike;
+  mode: string;
+  modeStatus: string;
+  autoReturn: BooleanLike;
+  autoPickup: BooleanLike;
+  reportDelivery: BooleanLike;
+  destination: string | null;
+  destinationsList: string[];
+  homeDestination: string | null;
+  botId: string;
+  allowPossession: BooleanLike;
+  possessionEnabled: BooleanLike;
+  paiInserted: BooleanLike;
+};
+
+const MuleControls = (props) => {
+  const { act, data } = useBackend<Data>();
+  const {
+    load,
+    autoReturn,
+    autoPickup,
+    reportDelivery,
+    destination,
+    homeDestination,
+    botId,
+    allowPossession,
+    possessionEnabled,
+    paiInserted,
+    destinationsList = [],
+  } = data;
+
+  return (
+    <>
+      <Section
+        title="Controls"
+        buttons={
+          <>
+            {!!load && (
+              <Button icon="eject" onClick={() => act('unload')}>
+                Unload
+              </Button>
+            )}
+            {!!paiInserted && (
+              <Button icon="eject" onClick={() => act('eject_pai')}>
+                Eject PAI
+              </Button>
+            )}
+          </>
+        }
+      >
+        <LabeledList>
+          <LabeledList.Item label="ID">
+            <Button onClick={() => act('setid')}>{botId}</Button>
+          </LabeledList.Item>
+          <LabeledList.Item label="Home">
+            <Button onClick={() => act('sethome')}>{homeDestination}</Button>
+          </LabeledList.Item>
+          <LabeledList.Item label="Destination">
+            <Dropdown
+              over
+              selected={destination || 'None'}
+              options={destinationsList}
+              width="188px"
+              onSelected={(value) => act('destination', { value })}
+            />
+          </LabeledList.Item>
+        </LabeledList>
+      </Section>
+      <Section title="Settings">
+        <Button.Checkbox checked={autoReturn} onClick={() => act('autored')}>
+          Auto-Return
+        </Button.Checkbox>
+        <br />
+        <Button.Checkbox checked={autoPickup} onClick={() => act('autopick')}>
+          Auto-Pickup
+        </Button.Checkbox>
+        <br />
+        <Button.Checkbox checked={reportDelivery} onClick={() => act('report')}>
+          Report-Delivery
+        </Button.Checkbox>
+        <br />
+        {allowPossession && (
```

Comment:
booleanlike will show 0 when its off
```suggestion
        {!!allowPossession && (
```